### ISSUE TITLE
Fix deprecated assertEquals in SyncEtagsUtilsTest in Services

### ISSUE THREAD

**Issue by wbrunette** [opendatakit/tool-suite-X#481]
Fix deprecated assertEquals in SyncEtagsUtilsTest in Services

"import static junit.framework.Assert.assertEquals" has been deprecated and should use the newer versions of assertEquals.

**Comment by Chinex-Boroja** [comment]
Can this issue be assigned to me @wbrunette ?

**Comment by Redeem-Grimm-Satoshi** [comment]
@Chinex-Boroja Can you finish first with the issue you have been assigned to?

**Comment by Chinex-Boroja** [comment]
@Redeem-Grimm-Satoshi done with it. Raised a PR already

**Comment by Redeem-Grimm-Satoshi** [comment]
It needs to be merged first before you can take on another one, Your two PRs are still open!

**Comment by Chinex-Boroja** [comment]
Okay noted. Thank you @Redeem-Grimm-Satoshi

**Comment by wbrunette** [comment]
We allow you to have one with a PR and a max of two others. You currently have 3 issues assigned to you, one has a PR leaving you at 2 issues actively working on. To spread the work, we prefer to let everyone have two active issues. If you would like to be swapped, please let us know what issue you want to be unassigned to.

**Comment by Stryker101** [comment]
Can this issue be assigned to me @wbrunette  @Redeem-Grimm-Satoshi My previous PR just got merged.

**Comment by siva224513** [comment]
can you please assign this issue to me @wbrunette @Redeem-Grimm-Satoshi

**Comment by maha-sachin** [comment]
Hi @wbrunette and @Redeem-Grimm-Satoshi, I am Maha from Outreachy. Could you please help me assign this issue to myself so I can start working on it? I'm eager to contribute and learn more about fixing deprecated assertions.

**Comment by wbrunette** [comment]
I have assigned the issue to @Stryker101

**Comment by Stryker101** [comment]
@wbrunette  @Redeem-Grimm-Satoshi  @Lamouresparus please review. thank you for your time.

**Comment by maha-sachin** [comment]
@wbrunette @Redeem-Grimm-Satoshi, could you please help me to assign a new issue? I'm eager to contribute and would appreciate your guidance in finding a suitable task to work on. Thank you!
